optimum/intel/generation/modeling.py

```diff
@@ -152,10 +264,10 @@ def forward(
             "attention_mask": attention_mask,
         }
 
+        nb_pkv = 2

```

Comment:
only used when `self.use_cache` is `True` and `past_key_values` is `None` so it might make sense to not declare both there

Reply:
done